src/solvers/strings/string_constraint_generator_format.cpp

```diff
@@ -363,21 +365,30 @@ exprt format_arg_from_string(const array_string_exprt &string,
         bitor_exprt{shl_exprt{typecast_exprt{string[2], type}, 16},
                     typecast_exprt{string[3], type}}};
   }
+
+  const and_exprt is_null{
+      equal_exprt{string[0], from_integer('n', string[0].type())},
+      equal_exprt{string[1], from_integer('u', string[0].type())},
+      equal_exprt{string[2], from_integer('l', string[0].type())},
+      equal_exprt{string[3], from_integer('l', string[0].type())},
+  };
```

Comment:
:no_entry_sign: I think there's a missing constraint on the length of the string.